.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!--
+- Please ensure this PR contains a single or a list of atomic commits.
+- At least one commit must have a message (What) and a description (Why).
+- Evey commit should have a message (What) and a description (Why).
+- Evey PR must have a description.
+- As an example you can use git ci -m"What" -m"Why" to achieve the requirements above. GitHub automatically recognises the commit description (-m"Why") in single commit PRs and adds it as the PR description.
+- Use imperative mood for every commit.
+- See for more details https://git.kernel.org/pub/scm/git/git.git/tree/Documentation/SubmittingPatches?id=HEAD#n110.
```

Comment:
This also means that you want that people prefix the commit description with the area they are operating on? If so, it may make sense to copy the relevant sections and fill it with hypershift related examples. It may make it easier for people to commit changes with the good feeling that they are doing it right. Pointing to a doc of another project may make it less obvious about what exactly you care about.

Reply:
ok I dropped it, only referencing our contributing guide now.

Reply:
Yeah "imperative mood" isn't immediately obvious until you look at the kernel docs, so I agree maybe having an example of good practice may be clearer